packages/synapse-react-client/src/components/RecentPublicationsGrid/RecentPublicationsGrid.tsx

```diff
@@ -0,0 +1,176 @@
+    queryResultBundle,
+  )
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: isSmallScreen || isMediumScreen ? 'column' : 'row',
+        gap: isSmallScreen || isMediumScreen ? '38px' : '80px',
+      }}
+    >
+      <Grid
+        container
+        spacing={2}
+        sx={{
+          order: isSmallScreen || isMediumScreen ? 1 : 0,
+          flex: isSmallScreen || isMediumScreen ? '0 0 100%' : '3',
+        }}
+      >
+        {dataRows.map(pub => (
+          <Grid item xs={12} sm={6} md={4} key={pub.rowId}>
+            <Card sx={{ height: '100%', boxShadow: 'none', border: 'none' }}>
+              <CardContent>
+                <Box display={'flex'} gap={'6px'} flexWrap={'wrap'}>
+                  {pub.values[tagColIndex] &&
+                    JSON.parse(pub.values[tagColIndex] as string).map(
+                      (tag: string, index: number) => (
+                        <Typography
+                          key={index}
+                          variant="overline"
+                          sx={{
+                            backgroundColor: '#EAECEE',
+                            borderRadius: '3px',
+                            padding: '4px 8px',
+                            border: 'none',
+                            lineHeight: 'initial',
+                          }}
+                        >
+                          {tag}
+                        </Typography>
+                      ),
+                    )}
+                </Box>
+                <Typography
+                  variant="headline2"
+                  sx={{
+                    padding: '20px 0px',
+                  }}
+                >
+                  {pub.values[titleColIndex]}
+                </Typography>
+                <Typography
+                  variant="body1"
+                  color="#71767F"
+                  sx={{
+                    fontStyle: 'italic',
+                    fontSize: '14px',
+                  }}
+                >
+                  {pub.values[journalColIndex]}
+                </Typography>
+                <Typography
+                  variant="body1"
+                  color="#71767F"
+                  sx={{ fontSize: '14px' }}
+                >
+                  {pub.values[publicationDateColIndex] &&
+                    formatDate(
+                      dayjs(Number(pub.values[publicationDateColIndex])),
+                      'MMMM, YYYY',
+                    )}
+                </Typography>
+              </CardContent>
+            </Card>
+          </Grid>
+        ))}
+      </Grid>
+
+      <Box
+        display="flex"
+        flexDirection="column"
+        gap="16px"
+        sx={{
+          height: '4px',
+          borderTop: '3px solid #DFE2E6',
+          alignItems: 'flex-start',
+          flex: isSmallScreen || isMediumScreen ? '0 0 100%' : '1',
+          padding: isSmallScreen || isMediumScreen ? '0 0 0 16px' : '0',
```

Comment:
```suggestion
          borderTop: '3px solid grey.400',
          alignItems: 'flex-start',
          flex: { xs: '0 0 100%', md: '1' },
          padding: { xs: '0 0 0 16px', md: '0' },
```

I can't remember if sx supports specifying the border color here. You might have to do:
```ts
          borderTop: '3px solid',
          borderColor: 'grey.400',
``